src/components/Posts/index.js

```diff
@@ -311,114 +311,111 @@ function Posts(props) {
     let postUrl = `/post/${props.note.id}?voteCount=${votesCount}`;
     return (
         <>
-            <div className="flex flex-col bg-black divide-y mt-2 overflow-hidden">
-                <div className="bg-gray-100 rounded-lg my-1 shadow-sm shadow-gray-400">
-                    <span className="flex p-2 text-black text-sm">
-                        <span className={'flex flex-grow font-medium'}>
-                            {truncateTitle(title, 100)}
-                        </span>
-                        <span className="flex justify-end text-gray-500 text-sm pr-1">
-                            {timeDifference.duration}
-                            {timeDifference.unit}{' '}
-                        </span>
-                    </span>
-                    <div className="flex justify-center py-2 px-1">
-                        {/*<Link to={postUrl}>*/}
+            <div className="flex flex-col items-center">
+                <div className="bg-white mt-2 mb-2 overflow-hidden rounded-lg w-full max-w-md">
+                    {/* Post Header: Title and Time */}
+                    <div className="p-3 border-b">
+                        <div className="flex justify-between items-center mb-2">
+                            <h3 className="text-sm font-semibold truncate">
+                                {truncateTitle(title, 20)}
```

Comment:
why are we updating the truncate length to 20 from 100?

Reply:
since we reduced the post size i optimized it to 20